src/Vehicle/Vehicle.cc

```diff
@@ -959,6 +966,11 @@ void Vehicle::_chunkedStatusTextCompleted(uint8_t compId)
             setPrearmError(messageText);
         }
     }
+    // check for autolog log downloading text
```

Comment:
Does the trigger button wait for download text from ardupilot? From the discussion it was decided not go ahead with this,

Reply:
it does not wait but still read the status text to show it on UI.